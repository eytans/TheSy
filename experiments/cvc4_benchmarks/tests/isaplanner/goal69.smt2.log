19:18:32 [WARN] Collecting statistics
19:18:32 [WARN]  ex is: zero
19:18:32 [WARN]  ex is: (succ zero)
19:18:32 [WARN]  ex is: (succ (succ zero))
19:18:32 [WARN]  ex is: nil
19:18:32 [WARN]  ex is: (cons autovar_Nat_0 nil)
19:18:32 [WARN]  ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:18:32 [WARN] Running exploration without proof mode, but goals were given
19:18:32 [WARN] Proof mode: true
19:18:32 [WARN]  ex is: zero
19:18:32 [WARN]  ex is: (succ zero)
19:18:32 [WARN]  ex is: (succ (succ zero))
19:18:32 [WARN]  ex is: nil
19:18:32 [WARN]  ex is: (cons autovar_Nat_0 nil)
19:18:32 [WARN]  ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:18:32 [WARN] Running exploration without proof mode, but goals were given
19:18:32 [WARN] Proof mode: true
19:18:32 [WARN] proved: (leq zero ?ts_ph_Nat_0) => true
19:18:32 [WARN] proved: (less ?ts_ph_Nat_0 ?ts_ph_Nat_0) => false
19:19:07 [WARN] proved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => true
19:19:07 [WARN] proved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => (less ?ts_ph_Nat_0 (succ ?ts_ph_Nat_0))
19:19:07 [WARN] proved: (less ?ts_ph_Nat_0 (succ ?ts_ph_Nat_0)) => (leq (succ zero) (succ ?ts_ph_Nat_0))
19:19:07 [WARN] proved: (less (succ ?ts_ph_Nat_0) ?ts_ph_Nat_0) => false
19:19:08 [WARN] proved: (less zero ?ts_ph_Nat_0) => (leq (succ zero) ?ts_ph_Nat_0)
19:19:08 [WARN] proved: (leq (succ zero) ?ts_ph_Nat_0) => (less zero ?ts_ph_Nat_0)
19:19:08 [WARN] proved: (delete ?ts_ph_Nat_1 (delete ?ts_ph_Nat_1 ?ts_ph_Lst_0)) => (delete ?ts_ph_Nat_1 ?ts_ph_Lst_0)
19:19:08 [WARN] proved: (delete ?ts_ph_Nat_1 ?ts_ph_Lst_0) => (delete ?ts_ph_Nat_1 (delete ?ts_ph_Nat_1 ?ts_ph_Lst_0))
19:19:08 [WARN] proved: (cons ?ts_ph_Nat_0 nil) => (delete (succ ?ts_ph_Nat_0) (cons ?ts_ph_Nat_0 nil))
19:19:08 [WARN] proved: (delete (succ ?ts_ph_Nat_0) (cons ?ts_ph_Nat_0 nil)) => (cons ?ts_ph_Nat_0 nil)
