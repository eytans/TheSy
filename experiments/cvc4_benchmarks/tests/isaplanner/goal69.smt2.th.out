19:18:32 [0m[33m[WARN] [0mCollecting statistics
(leq ?__x0 ?__y1) => (or (= ?__x0 ?__y1) (less ?__x0 ?__y1))
(or (= ?__x0 ?__y1) (less ?__x0 ?__y1)) => (leq ?__x0 ?__y1)
(less ?x zero) => false
(less zero (succ ?x)) => true
(less (succ ?x) (succ ?y)) => (less ?x ?y)
(less ?x ?y) => (less (succ ?x) (succ ?y))
(len nil) => zero
zero => (len nil)
(len (cons ?x ?y2)) => (succ (len ?y2))
(delete ?x nil) => nil
(delete ?x (cons ?y ?z)) => (ite (= ?x ?y) (delete ?x ?z) (cons ?y (delete ?x ?z)))
(ite (= ?x ?y) (delete ?x ?z) (cons ?y (delete ?x ?z))) => (delete ?x (cons ?y ?z))
Searcher: (less ?autovar5 (succ ?autovar1))
Searcher: (less ?autovar5 (succ ?autovar3))
Searcher: (less zero ?autovar6)
Searcher: (less (succ ?autovar4) ?autovar6)
19:18:32 [0m[33m[WARN] [0m ex is: zero
19:18:32 [0m[33m[WARN] [0m ex is: (succ zero)
19:18:32 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:18:32 [0m[33m[WARN] [0m ex is: nil
19:18:32 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_0 nil)
19:18:32 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:18:32 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:18:32 [0m[33m[WARN] [0mProof mode: true
19:18:32 [0m[33m[WARN] [0m ex is: zero
19:18:32 [0m[33m[WARN] [0m ex is: (succ zero)
19:18:32 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:18:32 [0m[33m[WARN] [0m ex is: nil
19:18:32 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_0 nil)
19:18:32 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:18:32 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:18:32 [0m[33m[WARN] [0mProof mode: true
19:18:32 [0m[33m[WARN] [0mproved: (leq zero ?ts_ph_Nat_0) => true
19:18:32 [0m[33m[WARN] [0mproved: (less ?ts_ph_Nat_0 ?ts_ph_Nat_0) => false
19:19:07 [0m[33m[WARN] [0mproved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => true
19:19:07 [0m[33m[WARN] [0mproved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => (less ?ts_ph_Nat_0 (succ ?ts_ph_Nat_0))
19:19:07 [0m[33m[WARN] [0mproved: (less ?ts_ph_Nat_0 (succ ?ts_ph_Nat_0)) => (leq (succ zero) (succ ?ts_ph_Nat_0))
19:19:07 [0m[33m[WARN] [0mproved: (less (succ ?ts_ph_Nat_0) ?ts_ph_Nat_0) => false
19:19:08 [0m[33m[WARN] [0mproved: (less zero ?ts_ph_Nat_0) => (leq (succ zero) ?ts_ph_Nat_0)
19:19:08 [0m[33m[WARN] [0mproved: (leq (succ zero) ?ts_ph_Nat_0) => (less zero ?ts_ph_Nat_0)
19:19:08 [0m[33m[WARN] [0mproved: (delete ?ts_ph_Nat_1 (delete ?ts_ph_Nat_1 ?ts_ph_Lst_0)) => (delete ?ts_ph_Nat_1 ?ts_ph_Lst_0)
19:19:08 [0m[33m[WARN] [0mproved: (delete ?ts_ph_Nat_1 ?ts_ph_Lst_0) => (delete ?ts_ph_Nat_1 (delete ?ts_ph_Nat_1 ?ts_ph_Lst_0))
19:19:08 [0m[33m[WARN] [0mproved: (cons ?ts_ph_Nat_0 nil) => (delete (succ ?ts_ph_Nat_0) (cons ?ts_ph_Nat_0 nil))
19:19:08 [0m[33m[WARN] [0mproved: (delete (succ ?ts_ph_Nat_0) (cons ?ts_ph_Nat_0 nil)) => (cons ?ts_ph_Nat_0 nil)
done in 36559
